prisma/schema.prisma

```diff
@@ -440,6 +442,8 @@ enum MobilityAid {
   MANUAL_CHAIR
   SCOOTER
   WALKER
+  CRUTCHES
+  OTHERS
```

Comment:
Added `CRUTCHES` and `OTHERS` to match the design

<img width="563" alt="image" src="https://user-images.githubusercontent.com/51224641/167759220-49db742d-a02a-4ae6-b112-b111efdd71c6.png">